lorien/Assets/I18n/en.txt

```diff
@@ -137,3 +137,30 @@ SAVE            Save
 DISCARD         Discard
 CANCEL          Cancel
 DELETE          Delete
+
+# -----------------------------------------------------------------------------
+# Action names
+# -----------------------------------------------------------------------------
+
+ACTION_shortcut_save_project       Save project
+ACTION_shortcut_new_project        New Project
+ACTION_shortcut_open_project       Open Project
+ACTION_shortcut_undo               Undo
+ACTION_shortcut_redo               Redo
+ACTION_shortcut_brush_tool         Brush tool
+ACTION_shortcut_line_tool          Line tool
+ACTION_shortcut_eraser_tool        Eraser tool
+ACTION_shortcut_select_tool        Selection tool
+ACTION_shortcut_move_tool          Move tool
+ACTION_shortcut_rectangle_tool     Rectangle tool
+ACTION_shortcut_circle_tool        Circle tool
+ACTION_shortcut_export_project     Export project
+ACTION_deselect_all_strokes        Deselect all strokes
+ACTION_center_canvas_to_mouse      Space
+ACTION_delete_selected_strokes     Delete selected strokes
+ACTION_copy_strokes                Copy
+ACTION_paste_strokes               Paste
+ACTION_duplicate_strokes           Duplicate strokes
+ACTION_toggle_distraction_free_mode Toggle distraction free mode
+ACTION_toggle_player               EFF TWELVE
```

Comment:
Maybe a bit of a "meme" translation. Still, might spike interest in what the key actually is doing :stuck_out_tongue:

Reply:
That's fine 😄